src/services/emptyS3Bucket.ts

```diff
@@ -0,0 +1,59 @@
+import {
+  DeleteObjectCommand,
+  GetBucketVersioningCommand,
+  PutBucketVersioningCommand,
+  S3Client
+} from '@aws-sdk/client-s3'
+import { getEnv } from '../utils/helpers'
+import { listS3Objects } from './listS3Objects'
+import { listS3ObjectVersions } from './listS3ObjectVersions'
+
+export const emptyS3Bucket = async (bucketName: string): Promise<void> => {
+  const s3Client = new S3Client({ region: getEnv('AWS_REGION') })
```

Comment:
This function looks a bit long to me, could it be split up? Also, there's a very long block inside an `if`, wonder if that could be made clearer?